camerad/archon.h

```diff
@@ -111,7 +113,7 @@ namespace Archon {
         std::vector<int> offset; //!< digital CDS offset (from TAPLINE definition)
         bool modeselected; //!< true if a valid mode has been selected, false otherwise
         bool firmwareloaded; //!< true if firmware is loaded, false otherwise
-        bool is_longexposure; //!< true for long exposure mode (exptime in sec), false for exptime in msec
+        bool longexposure_set; //!< true for long exposure mode (exptime in sec), false for exptime in msec
```

Comment:
having bools named is_... is a good convention, maybe it can be is_longexposure_set